.ci.cambpm

```diff
@@ -1,2 +1,2 @@
 @Library("camunda-ci") _
-buildMavenAndDeployToMavenCentral([jdk:8, extraJdks: ['jdk-11-latest'], mvn:3.5, licenseCheck:true, publishZipArtifactToCamundaOrg:true])
+buildMavenAndDeployToMavenCentral([jdk:11, extraJdks: ['jdk-17-latest'], mvn:3.5, licenseCheck:true, publishZipArtifactToCamundaOrg:true])
```

Comment:
:wrench: Since the Maven build is failing in Jenkin with JDK 17, we could remove the JDK 17 run to avoid this check affecting other PRs (until I removed the Jenkins CI).

```suggestion
buildMavenAndDeployToMavenCentral([jdk:11, mvn:3.5, licenseCheck:true, publishZipArtifactToCamundaOrg:true])
```

Reply:
❓ Should/can we also move to Maven 3.8 or will this be an issue?

Reply:
Moving to Maven 3.8 should be no problem for GitHub actions. 

But maybe, it is not supported in our Jenkins CI. If this is the case, we should dump Maven after removing the Jenkins CI. :wink:

Reply:
> Should/can we also move to Maven 3.8 or will this be an issue?

I didn't bump Maven as `feel-scala` is using the shade plugin and I have not checked yet if Maven 3.8 is a problem for it.